CODEOWNERS

```diff
@@ -139,17 +139,17 @@ Kconfig*                                  @tejlmand
 /samples/bluetooth/peripheral_fast_pair/  @MarekPieta @kapi-no @KAGA164
 /samples/bootloader/                      @hakonfam @oyvindronningstad
 /samples/matter/                          @Damian-Nordic @kkasperczyk-no
-/samples/crypto/                          @frkv @Vge0rge
+/samples/crypto/                          @frkv @Vge0rge @vili-nordic @joerchan @SebastianBoe @mswarowsky
 /samples/debug/ppi_trace/                 @nordic-krch @anangl
 /samples/hw_id/                           @maxd-nordic
 /samples/edge_impulse/                    @pdunaj
 /samples/esb/                             @lemrey
 /samples/app_event_manager/               @pdunaj @MarekPieta
 /samples/event_manager_proxy/             @rakons
 /samples/gazell/                          @leewkb4567
-/samples/keys/random_hw_unique_key/       @oyvindronningstad @Vge0rge
-/samples/keys/identity_key_generation/    @frkv @Vge0rge
-/samples/keys/identity_key_usage/         @frkv @Vge0rge
+/samples/keys/random_hw_unique_key/       @frkv @Vge0rge @vili-nordic @joerchan @SebastianBoe @mswarowsky
+/samples/keys/identity_key_generation/    @frkv @Vge0rge @vili-nordic @joerchan @SebastianBoe @mswarowsky
```

Comment:
Suggest to reduce this to just '/samples/keys/'

Reply:
done